scripts/build-anatomy-blank.py

```diff
@@ -72,9 +72,16 @@ def main():
 
     print(f"Loaded {len(id_list)} IDs from {args.id_list}",
           file=sys.stderr)
-
+          
+          
+    # filter by ids with a page in the portal
+    id_pages = cfde_common.get_portal_page_ids(term)
+    id_list_filtered = [value for value in id_list if value in id_pages]        
```

Comment:
this looks good!

you could also use sets -

```
id_list_filtered = set(id_list).intersection(set(id_pages))
```
which would do the for loop faster.